frontend/packages/ux-editor-v3/src/components/config/Expressions/Expressions.test.tsx

```diff
@@ -3,11 +3,10 @@ import { screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import type { ServicesContextProps } from 'app-shared/contexts/ServicesContext';
 import { renderWithMockStore } from '../../../testing/mocks';
```

Comment:
Alias can be added on line 5, 6, and 16 😊 
```suggestion
import { renderWithMockStore } from '@studio/testing/mocks';
```

Reply:
Same here as in [my previous comment](https://github.com/Altinn/altinn-studio/pull/12797#discussion_r1611766890)